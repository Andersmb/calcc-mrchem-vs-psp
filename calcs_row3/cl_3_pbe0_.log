Starting job 5688113 on c1-[21,26],c2-7,c3-[6,13],c5-7 at Thu May 19 22:54:47 CEST 2022

Copying cl_3_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/singlets
Copying cl_3_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/singlets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688113      cl_3_pbe0_        180                                             00:31:46      0:0 
5688113.bat+      batch         24        1   00:00:03          0   00:00:03   00:31:46      0:0 
5688113.ext+     extern        180        6   00:00:00          4   00:00:00   00:31:46      0:0 
5688113.0      mrchem.x        180       30   00:31:04         29   01:24:59   00:31:16      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688113                                                                          
5688113.bat+     19340K          0     19340K        0              0          0 
5688113.ext+          0          4          0        0              4          0 
5688113.0      9149328K         29 4027604855        0              0          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688113                                                                                               
5688113.bat+       19.44M               0         19.44M        0.07M                0          0.07M 
5688113.ext+        0.00M               4          0.00M            0                4              0 
5688113.0           6.75M               1          6.73M       67.01M               17         47.05M 

Job 5688113 completed at Thu May 19 23:26:33 CEST 2022
