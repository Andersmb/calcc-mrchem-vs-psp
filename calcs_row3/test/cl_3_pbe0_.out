                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Cl            :        0.000000        0.000000       -1.073603
    1       F            :        0.000000        0.000000        2.003438
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              13
 Beta electrons          :                                              13
 Total electrons         :                                              26
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999999511257e-01
    1    1    a          :                              9.999999803157e-01
    2    1    a          :                              9.999999398951e-01
    3    1    a          :                              1.000000018125e+00
    4    1    a          :                              1.000000046186e+00
    5    1    a          :                              1.000000000508e+00
    6    1    a          :                              1.000000297634e+00
    7    1    a          :                              1.000001118951e+00
    8    1    a          :                              9.999986618207e-01
    9    1    a          :                              1.000000111583e+00
   10    1    a          :                              1.000000098783e+00
   11    1    a          :                              9.999998671364e-01
   12    1    a          :                              9.999996522346e-01
   13    1    b          :                              9.999999277411e-01
   14    1    b          :                              9.999999803105e-01
   15    1    b          :                              9.999999398249e-01
   16    1    b          :                              1.000000018144e+00
   17    1    b          :                              1.000000046241e+00
   18    1    b          :                              1.000000000501e+00
   19    1    b          :                              1.000000297805e+00
   20    1    b          :                              1.000001143879e+00
   21    1    b          :                              9.999986637223e-01
   22    1    b          :                              1.000000111644e+00
   23    1    b          :                              1.000000114363e+00
   24    1    b          :                              9.999998729446e-01
   25    1    b          :                              9.999998806222e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                555.187741034648
 E-N energy              :            (au)              -1425.345906658091
 Coulomb energy          :            (au)                301.910105414078
 Exchange energy         :            (au)                 -9.389459224418
 X-C energy              :            (au)                -29.012706169180
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 49.723097141823
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -606.650225602962
 Nuclear energy          :            (au)                 49.723097141823
---------------------------------------------------------------------------
 Total energy            :            (au)             -5.569271284611e+02
                         :      (kcal/mol)             -3.494770466523e+05
                         :        (kJ/mol)             -1.462211964142e+06
                         :            (eV)             -1.515475853735e+04
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)               -101.111728142509
    1    1    a          :            (au)                -24.593951789858
    2    1    a          :            (au)                 -9.560885088029
    3    1    a          :            (au)                 -7.313132279554
    4    1    a          :            (au)                 -7.293222675950
    5    1    a          :            (au)                 -7.292792058316
    6    1    a          :            (au)                 -1.228254287537
    7    1    a          :            (au)                 -0.846931082560
    8    1    a          :            (au)                 -0.501141367621
    9    1    a          :            (au)                 -0.458629219162
   10    1    a          :            (au)                 -0.458156124137
   11    1    a          :            (au)                 -0.320662138374
   12    1    a          :            (au)                 -0.320570178986
   13    1    b          :            (au)               -101.111763372241
   14    1    b          :            (au)                -24.593952015794
   15    1    b          :            (au)                 -9.560885226007
   16    1    b          :            (au)                 -7.313132203095
   17    1    b          :            (au)                 -7.293222877505
   18    1    b          :            (au)                 -7.292792733297
   19    1    b          :            (au)                 -1.228256275563
   20    1    b          :            (au)                 -0.846928102594
   21    1    b          :            (au)                 -0.501139943766
   22    1    b          :            (au)                 -0.458634450959
   23    1    b          :            (au)                 -0.458158076592
   24    1    b          :            (au)                 -0.320660900406
   25    1    b          :            (au)                 -0.320576273895
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -322.600158884306
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          5.099020e+00        -556.927128461139         -5.569271e+02
    1          2.455876e-01        -559.679861793628         -2.752733e+00
    2          9.496025e-02        -559.769772996643         -8.991120e-02
    3          6.128245e-02        -559.784020869681         -1.424787e-02
    4          3.168303e-02        -559.787935653493         -3.914784e-03
    5          3.371635e-02        -559.790938035876         -3.002382e-03
    6          1.253993e-02        -559.791362956868         -4.249210e-04
    7          1.594569e-02        -559.791801962319         -4.390055e-04
    8          6.249324e-03        -559.791876351882         -7.438956e-05
    9          7.430979e-03        -559.791926307743         -4.995586e-05
   10          3.400866e-03        -559.791944241124         -1.793338e-05
   11          4.452459e-03        -559.791965784330         -2.154321e-05
   12          2.035145e-03        -559.791970915459         -5.131129e-06
   13          1.169964e-03        -559.791973252561         -2.337102e-06
   14          1.806893e-03        -559.791975808475         -2.555914e-06
   15          1.265884e-03        -559.791977222392         -1.413916e-06
   16          7.280340e-04        -559.791978003042         -7.806506e-07
   17          4.316215e-04        -559.791978234639         -2.315971e-07
   18          3.553230e-04        -559.791978423580         -1.889408e-07
   19          2.873356e-04        -559.791978514198         -9.061773e-08
   20          2.175698e-04        -559.791978562313         -4.811477e-08
   21          1.596950e-04        -559.791978591743         -2.943068e-08
   22          8.723681e-05        -559.791978603959         -1.221554e-08
---------------------------------------------------------------------------
                      SCF converged in 22 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Cl            :        0.000000        0.000000       -1.073603
    1       F            :        0.000000        0.000000        2.003438
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                558.929259606102
 E-N energy              :            (au)              -1432.039085916773
 Coulomb energy          :            (au)                302.000499513286
 Exchange energy         :            (au)                 -9.379335867470
 X-C energy              :            (au)                -29.026413080928
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 49.723097141823
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -609.515075745782
 Nuclear energy          :            (au)                 49.723097141823
---------------------------------------------------------------------------
 Total energy            :            (au)             -5.597919786040e+02
                         :      (kcal/mol)             -3.512747672442e+05
                         :        (kJ/mol)             -1.469733627104e+06
                         :            (eV)             -1.523271507769e+04
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)               -101.792189849074
    1    1    a          :            (au)                -24.830223364315
    2    1    a          :            (au)                 -9.611860326454
    3    1    a          :            (au)                 -7.351888369425
    4    1    a          :            (au)                 -7.335314974208
    5    1    a          :            (au)                 -7.335314943545
    6    1    a          :            (au)                 -1.281293239741
    7    1    a          :            (au)                 -0.860705131749
    8    1    a          :            (au)                 -0.559414077674
    9    1    a          :            (au)                 -0.513377441507
   10    1    a          :            (au)                 -0.513377438170
   11    1    a          :            (au)                 -0.348586844454
   12    1    a          :            (au)                 -0.348586815067
   13    1    b          :            (au)               -101.792189850260
   14    1    b          :            (au)                -24.830223363891
   15    1    b          :            (au)                 -9.611860327247
   16    1    b          :            (au)                 -7.351888369861
   17    1    b          :            (au)                 -7.335314975049
   18    1    b          :            (au)                 -7.335314944093
   19    1    b          :            (au)                 -1.281293239339
   20    1    b          :            (au)                 -0.860705133956
   21    1    b          :            (au)                 -0.559414083691
   22    1    b          :            (au)                 -0.513377440969
   23    1    b          :            (au)                 -0.513377437475
   24    1    b          :            (au)                 -0.348586844529
   25    1    b          :            (au)                 -0.348586814844
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -325.364265640589
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :        0.000000        0.000005       -0.103127
 Magnitude               :            (au)                        0.103127
                         :         (Debye)                        0.262124
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000       -0.220308
 Magnitude               :            (au)                        0.220308
                         :         (Debye)                        0.559967
---------------------------------------------------------------------------
 Total vector            :        0.000000        0.000005       -0.323435
 Magnitude               :            (au)                        0.323435
                         :         (Debye)                        0.822091
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 31m  5s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
